### PR TITLE
Calix (occamos) b6_316 and Calix (calix) 700

### PR DESCRIPTION
**Pull request by PipoCanaja**
Adding support for a few sensors on CALIX B6 (occamos)
Adding support for Calix 700 (calix)

If you would like to test this pull request then please run: 
`./scripts/github-apply 12744`

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 12744`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: mibs/calix/VDSL-LINE-MIB [hunk 1]
```diff
@@ -1,2840 +0,0 @@
-
```
Comment (PipoCanaja) [inline mibs/calix/VDSL-LINE-MIB:1]: This is a duplicate of a standard mib

**Comment by murrant** [comment]
occam (b6) should be a separate OS I think.

**Comment by PipoCanaja** [comment]
> occam (b6) should be a separate OS I think.

It actually is. calix 700 is in LibreNMS "calix" OS and CalixB6 is in LibreNMS "occamos" OS.

**Comment by murrant** [comment]
Sorry, I think I mis-understood a bit.  However, I'm not so sure the P-Series ONTs are "calix" os, as far as I know, that is only the old calix E5-1xx/E5-2xx products.

**Comment by PipoCanaja** [comment]
Hum, on that part I can't help. What would you suggest then ? I only keep the B6 part in this PR and get rid of the 700 part ?

**Comment by murrant** [comment]
I have no problem merging the b6 part.  Need to think on the 700 bit.

**Review by murrant — APPROVED** [review]

**Pull request merged**